Bottom: f5c3d6f4c537e73e679e11bb747cf25492430357
Top:    e8ad7d836075ede75dd23fa72606e1073abfed89
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:17:47 +0000

action.c: Mark some strings as translateable in ActionImport()


---

diff --git a/src/action.c b/src/action.c
index 34d54e2..07b2752 100644
--- a/src/action.c
+++ b/src/action.c
@@ -7613,7 +7613,7 @@ ActionImport (int argc, char **argv, int x, int y)
       if (!ds)
 	{
 	  const char *as = AttributeGet (PCB, "import::disperse");
-	  ds = gui->prompt_for("Enter dispersion:", as ? as : "0");
+	  ds = gui->prompt_for(_("Enter dispersion:"), as ? as : "0");
 	}
       if (units)
 	{
@@ -7661,7 +7661,7 @@ ActionImport (int argc, char **argv, int x, int y)
 	}
       else
 	{
-	  Message ("Bad syntax for Import(setnewpoint)");
+	  Message (_("Bad syntax for Import(setnewpoint)"));
 	  return 1;
 	}
